test/e2e/specs/all-features-confirmation/errors/errors.spec.js

```diff
@@ -337,107 +289,107 @@ describe('Error related test cases,', function() {
+            browser.get(url);
+            chaisePage.waitForElement(element(by.css('.modal-dialog ')));
+        });
 
-          chaisePage.waitForElement(modalTitle).then(function(){
-              return modalTitle.getText();
-          }).then(function (text) {
-              expect(text).toBe(testParams.facetErrorstext.invalidPageCriteriaTitle, "Invalid Page Criteria error pop-up could not be opened!");
-              return modalActionBody.getText();
-          }).then(function (errorText) {
-              expect(errorText).toBe(testParams.facetErrorstext.invalidPageCriteriaBody, "Error action text did not match");
-              done();
-          }).catch(function(error) {
-              console.log(error);
-              done.fail();
-          });
-      });
+        it('After clicking back button initial page should appear', function(done){
+            chaisePage.recordPage.getErrorModalOkButton().then(function(btn){
+                return btn.click();
+            }).then (function (){
+                return browser.navigate().back();
+            }).then(function (){
+                return browser.driver.getCurrentUrl();
+            }).then (function(currentUrl) {
+                expect(currentUrl).toContain('id=269111', "The back button failed to go back to previous page.");
+                done();
+            }).catch(function(error) {
+                done.fail(error);
+            });
+        });
 
-      it('On click of OK button the page should reload the page without paging conditions', function(done){
-          chaisePage.recordPage.getErrorModalOkButton().then(function(btn){
-              btn.click();
-              return browser.driver.getCurrentUrl();
-          }).then (function(currentUrl) {
-             recordsetPage = pageTestUrl.slice(0, pageTestUrl.search('@'));
-             expect(currentUrl).toBe(recordsetPage, "The redirection to Recordset page failed");
-             done()
-          }).catch( function(err) {
-              console.log(err);
-              done.fail();
-          });
-      });
+    });
+
+    describe("Dismissible Error Modal when using Navbar Delete button", function(){
+
+        beforeAll(function() {
+            browser.ignoreSynchronization = true;
+            var url = browser.params.url + "/record/#" + browser.params.catalogId + "/" + testParams.schemaName + ":" + testParams.table_name +  "/id=2002";
+            browser.get(url);
+            deleteBtn  = chaisePage.recordPage.getDeleteRecordButton();
+        });
 
+        it('Error modal is dismissible in case of conflict/forbidden error', function(done){
+            chaisePage.waitForElement(deleteBtn).then(function(){
+                return deleteBtn.click();
+            }).then(function () {
+                return chaisePage.recordPage.getConfirmDeleteButton().click();
+            }).then (function() {
+                closeModal = chaisePage.recordEditPage.getModalCloseBtn();
```

Comment:
Before we close the error modal that pops up after the confirm delete modal, we should check that the title and error message are as we expect in this case.